src/routes/assistant/[assistantId]/+page.svelte

```diff
@@ -15,7 +15,7 @@
 
 	afterNavigate(({ from }) => {
 		if (!from?.url.pathname.includes("settings")) {
-			previousPage = from?.url.pathname || previousPage;
+			previousPage = from?.url.toString() || previousPage;
```

Comment:
this way, query params are preserved as well

Reply:
btw here is another thing to fix (when I change page it should not retain external params) with params: https://huggingface.co/chat/assistants?utm_source=bensbites&utm_medium=referral&utm_campaign=daily-digest-going-out-with-ai&p=1